addons/account_peppol/data/mail_templates_email_layouts.xml

```diff
@@ -8,22 +8,18 @@
             <xpath expr="//t[hasclass('o_signature')]" position="after">
                 <div id="peppol_advertisement" t-if="peppol_info" style="font-size: 13px;">
                     <t t-if="peppol_info['is_peppol_sent']">
-                        <p style="min-width: 590px;">
-                            PS: This invoice has also been <b style="color: $o-enterprise-action-color">sent on Peppol</b>.
-                        </p>
+                        <i class="text-muted" style="min-width: 590px;">
+                            This invoice has also been <b style="color: $o-enterprise-action-color">sent via Peppol</b>.
+                        </i>
                     </t>
-                    <t t-if="not peppol_info['is_peppol_sent']">
-                        <p style="min-width: 590px;">
-                            PS: <b style="color: $o-enterprise-action-color;">We did not send your invoice on Peppol.</b>
+                    <t t-if="not peppol_info['is_peppol_sent'] and peppol_info['company_on_peppol'] and not peppol_info['partner_on_peppol']">
+                        <i class="text-muted" style="min-width: 590px;">
+                            <t t-out="company.name"/> uses <a target="_blank" href="https://www.odoo.com/app/invoicing?utm_source=db&amp;utm_medium=email&amp;utm_campaign=einvoicing" style="color: $o-enterprise-color;">Odoo</a> to send invoices, but this one <b style="color: $o-enterprise-action-color;">could not be sent via Peppol</b>.
```

Comment:
What about extracting all those code that is changing every month and put it python-side only?
```py
<t t-esc="peppol_info['extra_mail_html_message']" t-if="peppol_info['extra_mail_html_message']"/>
```